Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM ubuntu:18.04
+
+ENV BROWSER=/browser \
+    LC_ALL=en_US.UTF-8
+
+COPY requirements.txt ml_core/requirements.txt
+
+RUN apt-get update && \
+    apt-get install -y --no-install-suggests --no-install-recommends \
+      ca-certificates locales libxml2 libxml2-dev gcc g++ wget \
```

Comment:
Can you please try without `libxml2 libxml2-dev` - the new bblfsh client is supposed to not need it

Reply:
@Guillemdb did you check?

Reply:
ok, we use the old client so it is the same requirements.